src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -2,18 +2,49 @@
 
 import core.basesyntax.Storage;
 
+@SuppressWarnings("unchecked")
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int INITIAL_CAPACITY = 10;
+    private static final int DEFAULT_INDEX = 0;
+    private static final int NUMBER_ONE = 1;
```

Comment:
Constant names must share the context with the one who reads the code. Name NUMBER_ONE doesn't tell me more than `1`. 
```suggestion
    private static final int INITIAL_INDEX = 1;
```